test/api/test_rest.py

```diff
@@ -810,6 +814,11 @@ def test_groups(self):
         data = self.hit('/api/groups/')
         self.assertEqual('mygroup', data['results'][0]['slug'])
 
+    def test_groups_slug_field_only(self):
+        data = self.hit('/api/groups/?fields=slug')
```

Comment:
how about a test with more than 1 field?

Reply:
@mwasilew wouldn't harm at all :P

Reply:
@mwasilew add the test requested.